playground/README.md

```diff
@@ -35,10 +35,93 @@ Private Keys
 (9) 0x2a871d0798f97d79848a013d4936a73bf4cc922c825d33c1cf7073dff6d409c6
 ```
 
-Now with Rabby configured, and the services started, you can browse to http://localhost:8000 and make a trade with CoW
-Swap. Initially you will start with 10000 ETH, so you will have to wrap some ETH, or alternatively just simply test out
-an EthFlow order! You can follow along with watching the logs of the `autopilot`, `driver`, and `baseline` solver to see
-how the Protocol interacts.
+Now with Rabby configured, and the services started, you can browse to http://localhost:8000 and make a trade with CoW Swap.
+
+> Initially you will start with 10000 ETH, to run proper transaction you will need to wrap some ETH first.
+> The EthFlow is not configured by default, the next section explains how to set it up.
+> You can follow along with watching the logs of the `autopilot`, `driver`, and `baseline` solver to see how the Protocol interacts.
+
+## Setting up EthFlow
```

Comment:
TBH I'd remove all the ethflow documentation stuff again. Especially with this PR the playground is set up to work out of the box. No need to make it look like people have to read pages to make this work.